hw/ip/otbn/dv/model/iss_wrapper.cc

```diff
@@ -227,11 +242,21 @@ ISSWrapper::ISSWrapper() : tmpdir(new TmpDir()) {
   // drop all the fds when it execs.
   int fds[4];
   for (int i = 0; i < 2; ++i) {
+#ifdef __MACH__
+    if (pipe(fds + 2 * i)) {
+      std::ostringstream oss;
+      oss << "Failed to open pipe " << i << " for ISS: " << strerror(errno);
+      throw std::runtime_error(oss.str());
+    }
+    fcntl(fds[2 * i], F_SETFD, FD_CLOEXEC);
+    fcntl(fds[2 * i + 1], F_SETFD, FD_CLOEXEC);
+#else
     if (pipe2(fds + 2 * i, O_CLOEXEC)) {
       std::ostringstream oss;
       oss << "Failed to open pipe " << i << " for ISS: " << strerror(errno);
       throw std::runtime_error(oss.str());
     }
+#endif
```

Comment:
Can you just switch the code over to using `pipe()` unconditionally? That should work for Linux too, and avoids an ifdef. (Maybe add a comment like "You might think we could use pipe2 here, but that doesn't work for Macs" or similar, to avoid someone breaking it again in future).

Reply:
Updated